.github/workflows/tasklist-ci.yml

```diff
@@ -221,22 +221,3 @@ jobs:
           name: a11y-report
           path: tasklist/client/playwright-report/
           retention-days: 30
-
-  test-summary:
-    # Used by the merge queue as a dummy instead of the Zeebe CI in case only Operate files got changed:
-    # https://github.com/orgs/community/discussions/26251
-    # This name is hard-coded in the branch rules; remember to update that if this name changes
-    name: Test summary
-    if: always()
-    runs-on: ubuntu-latest
-    needs:
-      - run-build
-      - check_changes
-      - fe-type-check
-      - fe-eslint
-      - fe-tests
-      - fe-visual-regression-tests
-      - fe-a11y-tests
-      - fe-stylelint
-    steps:
-      - run: exit ${{ ((needs.check_changes.outputs.has_changed_frontend == 'true' && contains(needs.*.result, 'skipped') || contains(needs.*.result, 'failure')) && 1) || 0 }}
```

Comment:
Removing this will increase the likelihood of breaking changes introduced by Renovate in the Tasklist frontend.

Reply:
Are you sure?

To me this looks like a hack strapped upon our initial workaround of faking the "Test summary" status check of Zeebe CI. Comparing with https://github.com/camunda/camunda/blob/main/.github/workflows/operate-ci.yml#L85-L93 this fake "Test summary" job was initially without any dependencies or condition logic.

From Git history I can see that this logic was later added and if it is vital to the CI, it shouldn't be added to the same job that fakes the Zeebe CI for the required status check as that would be mixing multiple concerns...

Reply:
Yes, unfortunately this is a workaround built on top of a workaround, to prevent Renovate breaking FE dependencies upgrades that were happening very frequently.

Reply:
Regarding the breaking FE dependency upgrades I'd like to understand if that would be still a frequent (returning) problem after merging this PR.

Looking [at the list of currently open Renovate PRs affecting Tasklist](https://github.com/camunda/camunda/pulls?q=is%3Aopen+is%3Apr+author%3Aapp%2Frenovate+label%3Acomponent%2Ftasklist) none of them seems like FE dependency upgrade. That means that all FE dependency upgrades were either already auto-merged (aka Tasklist CI incl FE tests was green) or there were not that much.

Does this make sense? Given the current status that is a risk I'm willing to take and worst case prioritize https://github.com/camunda/camunda/issues/17721 even more as a follow-up if the problem returns.

Reply:
@cmur2 
> Looking [at the list of currently open Renovate PRs affecting Tasklist](https://github.com/camunda/camunda/pulls?q=is%3Aopen+is%3Apr+author%3Aapp%2Frenovate+label%3Acomponent%2Ftasklist) none of them seems like a FE dependency upgrade.

They were manually closed like this one https://github.com/camunda/camunda/pull/19235

>If Tasklist CI needs a status check we could either introduce a separate one like Zeebe's or wait until it is properly addressed in https://github.com/camunda/camunda/issues/17721

There is already a status check called "Tasklist CI test summary", but it is already returned by [tasklist-merge-ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/tasklist-merge-ci.yml). This one is also run in pull requests (for other reasons)
I think https://github.com/camunda/camunda/issues/17721 will be the answer to many of our problems

Reply:
@houssain-barouni are the Tasklist FE tests (or a relevant subpart of it) stable enough to be included in the https://github.com/camunda/camunda/blob/main/.github/workflows/tasklist-merge-ci.yml to prevent the dependency merges?

Reply:
yes, I did not observe any flakiness related to these tests, @vsgoulart can you confirm ?

Reply:
> yes, I did not observe any flakiness related to these tests, @vsgoulart can you confirm ?

@houssain-barouni @cmur2 Yes, all frontend tests included in this CI are all stable

Reply:
I just discussed with @houssain-barouni how to move on here since I agree that merging breaking Renovate changes is not desirable. That said, the current hack used by the Tasklist CI to re-use the fake status check of the Zeebe CI to ensure that is also not acceptable.

A better solution is to extend https://github.com/camunda/camunda/blob/main/.github/workflows/tasklist-merge-ci.yml (which runs on every PR + is a required status check of its own) to run 1 of the Tasklist frontend CI jobs (e.g. `fe-tests`) given that the job is stable enough to not cause flaky builds.

The implementation of this solution needs be done by the HTO team since it requires knowledge about which Tasklist frontend CI job is representative to avoid broken Renovate merges + this task is related to exclusively the Tasklist CI.

Reply:
I created https://github.com/camunda/camunda/issues/19428 (blocking this PR) for the refactoring mentioned above and to be taken care of by the HTO team.

Reply:
@vsgoulart @houssain-barouni now that https://github.com/camunda/camunda/issues/19428 is resolved, are you fine with merging this?